src/main/java/com/rabbitmq/client/impl/AMQCommand.java

```diff
@@ -152,6 +152,12 @@ public String toString(boolean suppressBody){
         }
     }
 
+    /**
+     * Instantiates and returns a StringBuilder
+     * @param body  Array of bytes that represents the content body
+     * @param suppressBody  A boolean that demonstrates if content body should be ignored
```

Comment:
This raises a question: what would be returned if the body is ignored. The answer is its length. If we are to document this method at all (not a part of the public API, hardly of much interest to most users and even contributors), we should make it reflect that.